packages/ai-jsx/src/stream/index.ts

```diff
@@ -80,10 +119,11 @@ export function toEventStream(renderable: Renderable): ReadableStream<StreamEven
  * content as SSE events.
  */
 export function toStreamResponse(renderable: Renderable): Response {
+  const renderResult = createRenderContext().render(renderable, { stop: () => false, map: (x) => x });
```

Comment:
`stop` is maybe unneeded here.

Reply:
Unforunately not because the presence of a `stop` is what indicates to `render` that the result should be an array of strings rather than a string. (See other comment about my regrets on the altitude of this API.)